.github/workflows/rpm-build-and-test.yml

```diff
@@ -1,22 +1,52 @@
 name: RPM Build and Test
 
 env:
+  # TODO: Surely this is a GH variable of some sort for the name of the repo
```

Comment:
`${{ github.event.repository.name }}` ??

Reply:
I tend to think that would be, say, `argobots` if the RPM Build and Test was being done as a downstream test of a PR for that repo which is why I was hesitant to use it and didn't have time to experiment with it.